modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-disable-individual-bootsource-update_{context}"]
+= Disabling an automatic update for a custom boot source
+
+You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for an individual custom boot source.
+
+.Procedure
+
+* Edit the `HyperConverged` custom resource (CR) and disable updates for a custom boot source by setting the `dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
```

Comment:
> and disable updates for a **custom** boot source by...

and disable updates for a **common** boot source by...

Reply:
The problem, as in the above comment, is we don't define "common boot sources" in our doc. So this depend on your answer to he above Thanks.

Reply:
Changed this content to reflect that we are speaking of boot sources that have NOT been customized. See new PR: https://github.com/openshift/openshift-docs/pull/48595